coding-guides/making-your-own-starboard.md

```diff
@@ -137,7 +137,7 @@ module.exports = class {
     const { starboardChannel } = this.client.settings.get(message.guild.id);
     const starChannel = message.guild.channels.find(channel => channel.name === starboardChannel)
```

Comment:
Good catch. Looks like there two instances of the guild channels not referencing the cache, though. Could you also update these? Line 221 and this one.